stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -429,6 +429,11 @@ public struct ${Self}<Pointee>
     return UnsafeMutablePointer(rawPtr)
   }
 
+  @available(*, unavailable, message: "Swift currently only supports freeing entire heap blocks, use deallocate() instead")
```

Comment:
Using `unavailable` in these changes will break existing code. Should this be something like `@available(swift, deprecated: 4.1, obsoleted: 5, message: "...")`? Do we have the machinery in place for those versions yet?